dev-util/sourcetrail/sourcetrail-2019.4.102.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="A free and open-source cross-platform source explorer"
+DESCRIPTION="Sourcetrail helps you get productive on unfamiliar source code"
+HOMEPAGE="https://www.sourcetrail.com/"
+COPYRIGHT="2019 Andreas Stallinger and others"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/CoatiSoftware/Sourcetrail/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="74b0f11904e5faaead94f92319001d77e23d102c7370f63f0f83a624e46312fe"
+SOURCE_DIR="sourcetrail-$portVersion"
+
+ARCHITECTURES="x86_64"
+SECONDARY_ARCHITECTURE="x86"
+
+PROVIDES="
+ 	cmd:sourcetrail
+	sourcetrail$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libQt5$secondaryArchSuffix >= 5.12
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Designer$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Network$secondaryArchSuffix
+	lib:libQt5Quick$secondaryArchSuffix
+	lib:libQt5Test$secondaryArchSuffix
+	lib:libQt5Xml$secondaryArchSuffix
+	lib:libboost_filesystem$secondaryArchSuffix
+	lib:libboost_iostreams$secondaryArchSuffix
+	lib:libboost_program_options$secondaryArchSuffix
+	lib:libboost_system$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libQt5$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Designer$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Network$secondaryArchSuffix
+	devel:libQt5Quick$secondaryArchSuffix
+	devel:libQt5Test$secondaryArchSuffix
+	devel:libQt5Xml$secondaryArchSuffix
+	devel:libboost_filesystem$secondaryArchSuffix
+	devel:libboost_iostreams$secondaryArchSuffix
+	devel:libboost_program_options$secondaryArchSuffix
+	devel:libboost_system$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:getconf
+	cmd:lrelease$secondaryArchSuffix >= 5
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	mkdir -p $prefix
+	cd $prefix
+	cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_BUILD_TYPE="Release" -DBOOST_ROOT=$portPackageLinksDir/devel~libboost_program_options/develop -DQt5_DIR=$portPackageLinksDir/devel~Qt/version/platform/compiler/lib/cmake/Qt5 $sourceDir
+}
+
+INSTALL()
+{
+	make Sourcetrail
+	mkdir -p $appsDir
+	mv $prefix/Sourcetrail $appsDir/Sourcetrail
+	addAppDeskbarSymlink $appsDir/Sourcetrail Sourcetrail
```

Comment:
Start with:
```diff
-	make Sourcetrail
-	mkdir -p $appsDir
-	mv $prefix/Sourcetrail $appsDir/Sourcetrail
-	addAppDeskbarSymlink $appsDir/Sourcetrail Sourcetrail
+	cd build
+	make install
```

Reply:
according to Sourcetrail document, there is no make install but make Sourcetrail

Reply:
Trust me, there is ...

Reply:
In the generated Makefile there is (here) at line10640 @echo "... install"

Reply:
Although there is a "make install" it fails to install, you should copy the needed files to $appsdir/Sourcetrail